src/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+div {
```

Comment:
краще стилізувати через класи, а не теги

Reply:
```suggestion
.block {
```
створи клас, та додай його необхідним елементам